### PR TITLE
Check for KVM and install kernel

### PR DESCRIPTION
**Pull request by enoch85**
cc @paulmenzel Your take on this?

### CODE REVIEW

File: nextcloud_install_production.sh [hunk 1]
```diff
@@ -896,6 +896,14 @@ then
     fi
 fi
 
+# Fix https://github.com/nextcloud/vm/issues/2018
+if [[ "$SYSVENDOR" == "QEMU" || "$SYSVENDOR" == "Red Hat" ]];
+then
+    # KVM
+    apt-get install -y --install-recommends \
+    linux-image-kvm
```
Comment (paulmenzel) [inline nextcloud_install_production.sh:904]: Do you know what recommends are that? I normally wouldn’t install them.

**Comment by paulmenzel** [comment]
I do not think, that it fixes issue https://github.com/nextcloud/vm/issues/2018, as that is about problems caused by installing *linux-image-kvm*.

**Comment by enoch85** [comment]
Hmm, but this adds kernel support for KVM right? 

I'm not on KVM myself, so you're the expert here.

**Comment by enoch85** [comment]
No feedback --> close.

**Pull request closed without merging**